### PR TITLE
[dagit] Load sensor + schedules in workspace context to avoid LeftNav query 

### PR DESCRIPTION
**Pull request by bengotow**
## Summary
When loading Dagit, the WorkspaceContext makes one query for repos + repo locations + jobs, and once it is ready the LeftNav mounts and requests the same data with the addition of sensors and schedules.

This consolidates the two queries  - we load LeftNav fragments about schedules and sensors (just a few fields) in the workspace query and it can be displayed immediately.  I think this should make the UI feel snappier since we're not stacking two very similar requests. Technically we're very close to also using the WorkspaceContext data for `useRepoSearch`, which fetches the same data a third time. I haven't changed that callsite yet.

This PR also removes a bunch of overfetching from the Instance > Sensors page. I think this dates back to when sensor statuses were shown on the main workspace landing page?

## Test Plan
Run Dagit, see that everything still loads!

## Checklist

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: js_modules/dagit/packages/core/src/instigation/Unloadable.tsx [hunk 1]
```diff
@@ -65,8 +65,10 @@ export const UnloadableSchedules: React.FunctionComponent<{
   }
   return (
     <>
-      <Subheading>Unloadable schedules</Subheading>
-      <UnloadableScheduleInfo />
+      <Box padding={{top: 16, horizontal: 24}}>
```
Comment (bengotow) [inline js_modules/dagit/packages/core/src/instigation/Unloadable.tsx:68]: Just noticed this was missing padding

File: js_modules/dagit/packages/core/src/workspace/RepositoryLocationsList.tsx [hunk 2]
```diff
@@ -17,12 +17,14 @@ import {Tooltip} from '../ui/Tooltip';
...
-import {WorkspaceContext} from './WorkspaceContext';
-import {RootRepositoriesQuery_workspaceOrError_Workspace_locationEntries as LocationOrError} from './types/RootRepositoriesQuery';
+import {WorkspaceContext, WorkspaceState} from './WorkspaceContext';
 
 const TIME_FORMAT = {showSeconds: true, showTimezone: true};
 
-const LocationStatus: React.FC<{location: string; locationOrError: LocationOrError}> = (props) => {
+const LocationStatus: React.FC<{
+  location: string;
+  locationOrError: WorkspaceState['locationEntries'][0];
```
Comment (hellendag) [inline js_modules/dagit/packages/core/src/workspace/RepositoryLocationsList.tsx:26]: Use the same type as the one referenced by `WorkspaceContext`? Using the `[0]` makes me feel a little itchy.

File: js_modules/dagit/packages/core/src/nav/FlatContentList.tsx [hunk 3]
```diff
@@ -187,7 +166,7 @@ const JobItem: React.FC<JobItemProps> = (props) => {
   );
 };
 
-const NAV_SCHEDULE_FRAGMENT = gql`
+export const NAV_SCHEDULE_FRAGMENT = gql`
```
Comment (hellendag) [inline js_modules/dagit/packages/core/src/nav/FlatContentList.tsx:169]: Is it still useful to have these as fragments, or can they just be part of the root workspace query?

**Review by hellendag — APPROVED** [review]

File: js_modules/dagit/packages/core/src/workspace/RepositoryLocationsList.tsx (hunk 2 above)
Comment (bengotow) [reply js_modules/dagit/packages/core/src/workspace/RepositoryLocationsList.tsx:26]: Yeah good call - I'll export a few more things from WorkspaceContext so it's easier to consume this!

File: js_modules/dagit/packages/core/src/nav/FlatContentList.tsx (hunk 3 above)
Comment (bengotow) [reply js_modules/dagit/packages/core/src/nav/FlatContentList.tsx:169]: Hmm that's a good question. Thought about this more and I think you're right, let's fold these away in favor of them just being part of the root workspace query.  I think there are a couple good reasons:

- Lots of other things use the WorkspaceContext and don't specify their data requirements

- It's tempting to add more things to these fragments without considering that they will be loaded immediately and always in the workspace context -- ideally we'd think about new fields carefully for the root query.

**Comment by bengotow** [comment]
Thanks for the feedback @hellendag! Will get this merged tonight 🙌

**Pull request merged**